guides/v2.2/extension-dev-guide/indexing.md

```diff
@@ -183,76 +183,25 @@ You can reindex by:
-<td />
-</tr>
-<tr>
-<td>Category products</td>
-<td><code>catalog_category_product</code></td>
-<td><a href="{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Catalog/Model/Indexer/Category/Product.php" target="_blank">Magento\Catalog\Model\Indexer\Category\Product</a></td>
-<td>Creates category/products association</td>
-</tr>
-<tr>
-<td>Product categories</td>
-<td><code>catalog_product_category</code></td>
-<td><a href="{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Catalog/Model/Indexer/Product/Category.php" target="_blank">Magento\Catalog\Model\Indexer\Product\Category</a></td>
-<td>Creates category/products association</td>
-</tr>
-<tr>
-<td>Product price</td>
-<td><code>catalog_product_price</code></td>
-<td><a href="{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Catalog/Model/Indexer/Product/Price.php" target="_blank">Magento\Catalog\Model\Indexer\Product\Price</a></td>
-<td>Pre-calculates product prices</td>
-</tr>
-<tr>
-<td>Product entity attribute value</td>
-<td><code>catalog_product_attribute</code></td>
-<td><a href="{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Catalog/Model/Indexer/Product/Eav.php" target="_blank">Magento\Catalog\Model\Indexer\Product\Eav</a></td>
-<td>Reorganizes the EAV product structure to flat structure</td>
-</tr>
-<tr>
-<td>Stock</td>
-<td><code>cataloginventory_stock</code></td>
-<td><a href="{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/CatalogInventory/Model/Indexer/Stock.php" target="_blank">Magento\CatalogInventory\Model\Indexer\Stock</a></td>
-<td />
-</tr>
-<tr>
-<td>Catalog rule product</td>
-<td><code>catalogrule_rule</code></td>
-<td><a href="{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/CatalogRule/Model/Indexer/Rule/RuleProductIndexer.php" target="_blank">Magento\CatalogRule\Model\Indexer\Rule\RuleProductIndexer</a></td>
-<td />
-</tr>
-<tr>
-<td>Catalog product rule</td>
-<td><code>catalogrule_product</code></td>
-<td><a href="{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/CatalogRule/Model/Indexer/Product/ProductRuleIndexer.php" target="_blank">Magento\CatalogRule\Model\Indexer\Product\ProductRuleIndexer</a></td>
-<td />
-</tr>
-<tr>
-<td>Catalog search</td>
-<td><code>catalogsearch_fulltext</code></td>
-<td><a href="{{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/CatalogSearch/Model/Indexer/Fulltext.php" target="_blank">Magento\CatalogSearch\Model\Indexer\Fulltext</a></td>
-<td />
-</tr>
-</tbody>
-</table>
+The {{site.data.var.ce}} application implements the following indexers (use [bin/magento indexer:info]({{ page.baseurl }}/reference/cli/magento.html#indexerinfo) to get the list of indexers):
```

Comment:
I agree with this. Last week, I asked a contributor to remove the output of the command that lists message queues, because the list is ever-expanding.

Reply:
**fixed**

@hguthrie , could you please review a final changes?

Thank you!